artemis-core/artemis/src/main/java/com/artemis/AspectSubscriptionManager.java

```diff
@@ -57,12 +79,42 @@ protected void setWorld(World world) {
      * See https://github.com/junkdog/artemis-odb/issues/551
 	 *
 	 * @param builder Aspect to match.
+	 * @param applyDefaults whether to apply default aspect
 	 * @return {@link EntitySubscription} for aspect.
 	 */
-	public EntitySubscription get(Aspect.Builder builder) {
+	EntitySubscription get(Aspect.Builder builder, boolean applyDefaults) {
```

Comment:
This flag was introducted because of EntityTransmuter. My assumption was that defaults should not apply for this class and I also wasn't a fan of mutating the Aspect passed to EntityTransmuter in AspectFieldResolver to achieve this behaviour.
If we keep this behaviour (through whatever means), we should document that defaults will never apply to EntityTransmuter in AspectDescriptor and Exclude.